ci/terraform/aws/api-gateway.tf

```diff
@@ -0,0 +1,3 @@
+resource "aws_api_gateway_rest_api" "di-authentication-api" {
+  name = format("di-authentication-api-%s", var.environment)
```

Comment:
Should probably use string interpolation here `"di-authentication-api-${var.environment`